apps/backend/package.json

```diff
@@ -21,8 +21,10 @@
   },
   "dependencies": {
     "@nestjs/common": "^10.0.0",
+    "@nestjs/config": "^3.1.1",
     "@nestjs/core": "^10.0.0",
     "@nestjs/platform-express": "^10.0.0",
+    "mongoose": "^7.5.2",
```

Comment:
Also mongoose was added accidentally, but I'll use it in the next commits related to db